src/frontend/e2e/page-objects/PhylogeneticTreePage.ts

```diff
@@ -0,0 +1,102 @@
+import {Locator, Page} from '@playwright/test'
+import {TreeInfo} from '../utils/schemas/treeInfo'
+
+export class PhylogeneticTreePage {
+
+    readonly page: Page
+   
+        readonly treeNameInput: Locator
+        readonly overviewRatioButton: Locator
+        readonly targetedRatioButton: Locator
+        readonly nonContextualizedRatioButton: Locator
+        readonly lineageDropDown: Locator
+        readonly collectionDateDropDown: Locator
+        readonly forcedIncludeTextArea: Locator
+        readonly addForcedSamplesButton: Locator
+        readonly createTreeButton: Locator
+        readonly radiosTreeType: Locator
+        readonly errorToolTip: Locator
+        readonly exceedNumberOfCharsErrorMsg: Locator
+        readonly sampleWarningMessage: Locator
+
+    constructor(page: Page){
+        this.page = page
+        this.treeNameInput = page.locator("#outlined-basic");
+        this.overviewRatioButton = page.locator("input[value='OVERVIEW']");
+        this.targetedRatioButton = page.locator("input[value='TARGETED']");
+        this.nonContextualizedRatioButton = page.locator("input[value='NON_CONTEXTUALIZED']");
+        this.lineageDropDown = page.locator("p + button[label='All']");
+        this.collectionDateDropDown = page.locator("p + button[label='All Time']");
+        this.forcedIncludeTextArea = page.locator("//div[contains(@class,'MuiDialog-paperScrollPaper')]/descendant::textarea[not(@aria-hidden='true')]");
```

Comment:
We should try to avoid using CSS selectors. Lets reach to devs to see how we can add test id ourselves or if they prefer to do this themselves

Reply:
if you two can add the test-ids yourself following charles' best practice guidelines that would be great 👍

Reply:
do you think if xpaths could work in the mean time?

Reply:
xpath is even worst that CSS. It dissects the whole DOM making tests very slow and on top that addition/removal/modification of elements causes changes in DOMS, hence Xpath